R/render.R

```diff
@@ -280,6 +290,42 @@ render_quarto_lang <- function(language_code, path, output_dir, type) {
 
 }
 
+#' Filter the freeze directory
+#'
+#' This function filters the freeze directory keeping only the files
+#' that match the language code.
+#'
+#' @param temporary_directory Temporary directory where the project is
+#' @param project_name Name of the project directory
+#' @param language_code The Language code for the current rendering
+#' @dev
```

Comment:
seeing docs for internal functions makes me so happy :face_holding_back_tears:

Reply:
I can work on that 😄 In my personal packages ALL my internal functions are documented for future me to remember what the hell I was trying to with a particular function 😅

Reply:
honestly before devtag I had trouble seeing the point of using roxygen2 for internal functions :sweat_smile: